apps/e2e/cypress/e2e/back-office-applications/addRepresentation.spec.js

```diff
@@ -33,15 +32,6 @@ describe('Add representation scenarios', () => {
 		searchResultsPage.clickLinkByText('Relevant representations');
 		searchResultsPage.clickButtonByText('Add a representation');
 		representationPage.fillRepresentationDetails();
-	});
-	it('As a user able to search representation', () => {
```

Comment:
is this not needed anymore?

Reply:
Not required  because download is included in another test